ivy/functional/frontends/numpy/random/functions.py

```diff
@@ -233,3 +233,14 @@ def triangular(left, mode, right, size=None):
         right - (right - mode) * ((1 - u) * (right - mode) / (right - left)) ** 0.5
     )
     return ivy.where(condition, values1, values2)
+
+
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+def zipf(a, size=None):
+    if a <= 1:
+        raise ValueError(
+            "Parameter 'alpha' must be greater than 1 for the Zipf distribution."
+        )
+    return ivy.floor((1.0 / a) ** 1.0).astype(int)
+
```

Comment:
Can you please state what's your reference for this implementation? Maybe a link to any doc/paper etc? I'm not sure if this is how the `zipf` distribution is calculated.

Reply:
Hey @hmahmood24 ,thanks for your review.                                                                                           This is the link to the documentation  on zipf function https://numpy.org/doc/stable/reference/random/generated/numpy.random.zipf.html                                                                                                          This is the Test for the zipf function  https://github.com/unifyai/ivy/pull/22061 which is also another PR. Advise on how I may merge them also if possible.

Reply:
i have had to push my commits under a new branch so as to fix all issues as advised. I however had to create a new pull request with the following link   https://github.com/unifyai/ivy/pull/22424
It contains both the zipf function and the test. Should i close this PR . Advise please